pkg/plugin/linuxutil/testdata/wrong-netstat

```diff
@@ -1 +1 @@
-Test: aa aa aa
+Test: aa aa aa
```

Comment:
Files should end with new lines. As far as I can tell from looking at this today, it is reasonable to expect /proc/net/netstat to always end with newlines, so I don't think changing the test is the right thing here.